static/elements/chromedash-app.js

```diff
@@ -129,23 +129,23 @@ class ChromedashApp extends LitElement {
     });
     page('/guide/edit/:featureId', (ctx) => {
```

Comment:
Do we want to change this to `'/guide/edit/:featureId(-?\d+)'`. 

I think this *should* work but I didn't try this out personally. There's talk about it on this [issue](https://github.com/visionmedia/page.js/issues/251). That issue redirects to this express test router [page](http://forbeslindesay.github.io/express-route-tester/) (since PageJS is inspired by Express)

| Scenario      | Screenshot | parseInt result |
| ----------- | ----------- | ----------- |
| positive numbers using (-?\d+) regex `'/guide/edit/2'`    | ![image](https://user-images.githubusercontent.com/7788930/191273641-79bbcc8e-9f3c-447a-885d-50ecd6dc3c8a.png)      |  2 ✅ |
| negative numbers using (-?\d+) regex `'/guide/edit/-2'`    | ![image](https://user-images.githubusercontent.com/7788930/191274077-ccde64d7-bc0f-4374-963e-1dc793e807e7.png)      |  -2 ✅ |
| 0 using (-?\d+) regex `'/guide/edit/0'`    | ![image](https://user-images.githubusercontent.com/7788930/191274267-43c8c83a-2169-4244-adf0-b7d70ff8a5f2.png)     |  0 ✅ |
| non integer using (-?\d+) regex `'/guide/edit/hi'`    | ![image](https://user-images.githubusercontent.com/7788930/191274485-96c4448e-cbf6-40db-babe-287464f40195.png)   |  N/A (Doesn't match this route)  ✅ |
| non integer without using (-?\d+) regex `'/guide/edit/hi'`    | ![image](https://user-images.githubusercontent.com/7788930/191274637-435bb156-a4e7-4f24-a4ef-603b0ca53239.png)   |  NaN  ❌ |

Why change it to this:
It will give us better safety for the route. Without it, a user could go to `/guide/edit/somethingThatsNotAInteger` and parseInt will return NaN.

*Note this regex also matches `-0` but that is okay because parseInt interprets that as a number too*

Reply:
Similar changes for the other routes too if we go with this.

Reply:
Good suggestion.  I was expected to get a NaN with the existing code as you said above, but somehow I couldn't get it to do it.  I went ahead and added the regexes because I think that they helot make our expectations about the URLs more explicit.  I left off the dash because all our IDs seem to be positive.

Also, I learned that while testing the SPA, one needs to look at the Network tab filtered by doc requests to verify that no new full page loads are happening.